README.md

```diff
@@ -136,6 +136,10 @@ selectively enabled or disabled:
   true if cljfmt should break hashmaps onto multiple lines. This will
   convert `{:a 1 :b 2}` to `{:a 1\n:b 2}`. Defaults to false.
 
+*`:remove-multiple-non-indenting-spaces?` - true if cljfmt should
```

Comment:
You're missing a space between the `*` and the \`. I'm not sure if the markdown parser will like that. Also can you use the same formatting as the other bullet points? So a newline after the `-`.

Reply:
should be good now